src/arch/e2k/e2k_syscall_handler.c

```diff
@@ -22,7 +22,7 @@ unsigned long cpuSyscall(int number, uint64_t arg1, uint64_t arg2,
 unsigned long cpuJmpSyscall(int number, uint64_t arg1, uint64_t arg2,
 			 uint64_t arg3, uint64_t arg4)
 {
-	printk("jmpsyscall handler %d (%x, %x, %x, %x)\n", number, arg1, arg2, arg3, arg4);
+	printk("jmpsyscall handler %d (%"PRIu64", %"PRIu64", %"PRIu64", %"PRIu64")\n", number, arg1, arg2, arg3, arg4);
```

Comment:
Why not just `%llx`? We usually don't use this "PRI.." stuff.

Reply:
It's more correctly using PRIu64 in printf() for uint64_t.
I think we should replace another than PRI for 'stdint' types in future